src/pfe/portal/routes/templates.route.js

```diff
@@ -86,31 +105,51 @@ router.delete('/api/v1/templates/repositories', validateReq, async (req, res, _n
     if (error instanceof TemplateError && error.code === 'REPOSITORY_DOES_NOT_EXIST') {
       res.status(404).send(error.message);
       return;
+    } else if (error instanceof TemplateError && error.code === 'LOCKED') {
```

Comment:
might be clearer to check instanceof once, and then then the error code inside that